package/build.gradle

```diff
@@ -0,0 +1,27 @@
+description = 'Sparkling Water Spark Package'
+
+apply plugin: 'com.github.johnrengelman.shadow'
+
+dependencies {
+    // hack to go around Spark Ivy resolver
+    compileOnly "no.priv.garshol.duke:duke:1.2"
+
+    compile project(':sparkling-water-core')
+    compile project(":sparkling-water-examples")
+    compile project(":sparkling-water-ml")
+    compile project(":sparkling-water-repl")
+}
+
+jar {
+    enabled = true
+}
+
+shadowJar {
+    archiveName = jar.archiveName
```

Comment:
@jakubhava The problem here is that we cannot shadow jetty without processing H2O dependencies. That would mean to include h2o dependencies and i would like to avoid that. 

But worth experimenting - we can include only shaddowed jetty and processed h2o-core, but we need to make sure, the we publish dependencies in POM correctly.

@tomkraljevic the problem is more complex. In the context of Spark package, there is no actual binary which you can put on `-cp` as it is mentioned in documentation.  There is only a reference to a package. However, users can do that with binary distribution from download page (we need to put it into SW documentation).

Furthermore, so far it looks like people mostly using automatic mode (equivalent of `-internal_security_enabled` in Hadoop).